packages/combo-box/theme/material/vaadin-combo-box-overlay-styles.js

```diff
@@ -1,5 +1,4 @@
 import '@vaadin/vaadin-material-styles/color.js';
-import '@vaadin/overlay/theme/material/vaadin-overlay.js';
```

Comment:
Same as above, this import it's no longer needed as the component does not define `vaadin-overlay`.